examples/with-mongodb-mongoose/lib/dbConnect.js

```diff
@@ -29,9 +29,6 @@ async function dbConnect() {
       useNewUrlParser: true,
       useUnifiedTopology: true,
       bufferCommands: false,
-      bufferMaxEntries: 0,
-      useFindAndModify: false,
-      useCreateIndex: true,
```

Comment:
It looks like all of these except `bufferCommands` can be removed per the docs here https://mongoosejs.com/docs/migrating_to_6.html#no-more-deprecation-warning-options if we update to the latest version of `mongodb` which is `v4.1.2` currently.